CHANGES_UNRELEASED.md

```diff
@@ -28,7 +28,8 @@ Use the template below to make assigning a version number during the release cut
 ## Nimbus ⛅️🔬🔭
 
 ### 🦊 What's Changed 🦊
-
+- Added a `stateful` cargo feature and added appropriate feature flag attributes ([#5448](https://github.com/mozilla/application-services/pull/5448)).
+  - This does not functionally change build processes, as the `stateful` feature is now the default feature for the `nimbus-sdk` library.
 - Changed the ordering around for optional arguments for Python compatibility ([#5460](https://github.com/mozilla/application-services/pull/5460)).
   - This does not change Kotlin or Swift APIs, but affects code that uses the uniffi generated FFI for `record_event` and `record_past_event` directly.
 - Refactor the `EnrollmentEvolver` in preparation for a larger refactor to split out a `stateful` feature. ([#5374](https://github.com/mozilla/application-services/pull/5374)).
```

Comment:
The reference to #5374 can probably be folded in to your log entry.

At a bare minimum it should happen occur chronologically before (and next to) your log entry.